plugins/linkattacher/src/linkattacherserverimpl.cpp

```diff
@@ -92,34 +92,34 @@ bool LinkAttacherServerImpl::attachUnscoped(const string& parent_model_name, con
     }
 
     //This is joint creation
-    gazebo::physics::JointPtr fixed_joint;
+    gazebo::physics::JointPtr joint;
 
     #if GAZEBO_MAJOR_VERSION >= 8
-    fixed_joint = _world->Physics()->CreateJoint("fixed",parent_model);
+    joint = _world->Physics()->CreateJoint(jointType,parent_model);
     #else
-    fixed_joint = _world->GetPhysicsEngine()->CreateJoint("fixed",parent_model);
+    joint = _world->GetPhysicsEngine()->CreateJoint("fixed",parent_model);
```

Comment:
Any reason why `fixed` is not changed to `jointType` here?

Reply:
leftover, fixed in https://github.com/robotology/gazebo-yarp-plugins/pull/461/commits/acd167ae506eae4492220c4fa4db6cdbcb718f79